custom_components/tuya_local/devices/smart_camera_sc116-wz3a.yaml

```diff
@@ -0,0 +1,511 @@
+name: Camera
+products:
+  - id: b60fclvud3xtcufm
+    manufacturer: Tuya
+    model: SC116-WZ3A
+entities:
+  - entity: camera
+    dps:
+      - id: 105
+        type: boolean
+        optional: true
+        name: switch
+        mapping:
+          - dps_val: true
+            value: false
+          - dps_val: false
+            value: true
+          - dps_val: null
+            value: true
+            hidden: true
+      - id: 134
+        type: boolean
+        name: motion_enable
+        optional: true
+      - id: 150
+        type: boolean
+        name: record
+        optional: true
+      - id: 115
+        type: base64
+        optional: true
+        persist: false
+        sensitive: true
+        name: snapshot
+  - entity: switch
+    name: Indicator
+    category: config
+    icon: "mdi:led-outline"
+    dps:
+      - id: 101
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Flip image
+    category: config
+    icon: "mdi:flip-vertical"
+    dps:
+      - id: 103
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Time OSD
```

Comment:
Normally "Watermark" is used, along with `icon: "mdi:watermark"`. It will make later translation expansion easier if a common name is used.